rllib/agents/slateq/slateq_torch_policy.py

```diff
@@ -41,14 +41,15 @@ def build_slateq_model_and_distribution(
     Returns:
         Tuple consisting of 1) Q-model and 2) an action distribution class.
     """
+    device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
```

Comment:
Actually, the TorchPolicy will take care of all this.

I think we only have to make sure all tensors in the loss function are on the right device. ...

Reply:
ok, I understand everything now.
it's the target_model that was the issue. we just need to use the correct target_model out of policy.target_models for things to work.
thanks.

Reply:
Ah, yeah, sorry, I should have thought about this. Yes, you can always do:
```
correct_target_model_to_use = policy.target_models[model]
...
```